### DISCUSSION TITLE
How can I change onboarding PR's target branch?

### DISCUSSION THREAD

**Discussion by sorrowfulwarrior** [renovatebot/renovate#20145] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.83.0-slim

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.6

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to test the renovate in a project but I don't want to break anything in the main branch and I also like to have a dashboard. Is there any way to change first PR's target branch?

### What you tried so far.

.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Renovate's config - if present - must always be in the default branch.

Possibility one:
- Set the `renovate.json` in default branch to be like `{ "baseBranches": ["dev"] }`. All subsequent PRs will target `dev` and not the main branch after onboarding

Possibility two:
- As above, but add `useBaseBranchConfig=true` in the default branch renovate config
- Set whatever you want in the `renovate.json` in the `dev` branch (it will extend and override the default branch config)

Possibility three (if you're just testing one repo):
- Set `requireConfig=optional, baseBranches=[dev], useBaseBranchConfig=true` in your bot global config
- No `renovate.json` in your default branch
- Add `renovate.json` to your `dev` branch.. but in this case it would essentially be the same as putting everything in the bot config

**Reply by sorrowfulwarrior** [reply]
Thank you for your fast response. I think for me the last option would work as I don't want to add anything to the default branch.
I also saw in discussions that it is possible to set `requireConfig=ignored`. If I set it to `ignored`, does renovate still use the specific `renovate.json` configs which are in sub repositories?

**Reply by rarkins** [reply]
https://docs.renovatebot.com/self-hosted-configuration/#requireconfig

**Reply by sorrowfulwarrior** [reply]
Thanks <3
